src/test/libcephfs/test.cc

```diff
@@ -655,6 +655,21 @@ TEST(LibCephFS, Xattrs_ll) {
   ceph_shutdown(cmount);
 }
 
+TEST(LibCephFS, check_dummy_op) {
+  struct ceph_mount_info *cmount;
+  ASSERT_EQ(ceph_create(&cmount, NULL), 0);
+  ASSERT_EQ(ceph_conf_read_file(cmount, NULL), 0);
+  ASSERT_EQ(0, ceph_conf_parse_env(cmount, NULL));
+  ASSERT_EQ(ceph_mount(cmount, NULL), 0);
+
+  UserPerm *perms = ceph_mount_perms(cmount);
+
+  int res = ceph_check_dummy_op(cmount, perms);
+  ASSERT_EQ(res, 0);
```

Comment:
0???

Reply:
I will change this up

Reply:
Changed to ` ASSERT_NE(res, 0)`

Reply:
Maybe you should do what the `Server::reclaim_session(Session *session, const cref_t<MClientReclaim> &m)` does just returning `-CEPHFS_EOPNOTSUPP` to client and then evict the client.

Reply:
Okay, in the `void Server::dispatch_client_request(MDRequestRef& mdr)` it has already returned a `-CEPHFS_EOPNOTSUPP` to client. 

BTW, should we evict the client here too ?

Reply:
@gregsfortytwo @batrick what are your thoughts on this?